packages/sit-onyx/src/components/OnyxAccordionItem/OnyxAccordionItem.vue

```diff
@@ -0,0 +1,154 @@
+<script lang="ts" setup>
+import chevronLeftSmall from "@sit-onyx/icons/chevron-left-small.svg?raw";
+import { computed, inject, useId } from "vue";
+import { ACCORDION_INJECTION_KEY, OnyxIcon, useDensity } from "../../";
+import { SKELETON_INJECTED_SYMBOL, useSkeletonContext } from "../../composables/useSkeletonState";
+import OnyxSkeleton from "../OnyxSkeleton/OnyxSkeleton.vue";
+import type { OnyxAccordionItemProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxAccordionItemProps>(), {
+  disabled: false,
+  skeleton: SKELETON_INJECTED_SYMBOL,
+});
+
+const itemId = useId();
+
+defineSlots<{
+  header(): unknown;
+  panel(): unknown;
+}>();
+
+const accordionContext = inject(ACCORDION_INJECTION_KEY);
+
+const isOpen = computed({
+  get: () => accordionContext?.openItems.value.has(itemId) || false,
+  set: (value: boolean) => {
+    accordionContext?.updateOpen(itemId, value);
+  },
+});
+
+const skeleton = useSkeletonContext(props);
+const { densityClass } = useDensity(props);
+</script>
+
+<template>
+  <div
+    v-if="skeleton || accordionContext?.skeleton.value"
+    :class="['onyx-component', 'onyx-accordion-item-skeleton', densityClass]"
+  >
+    <OnyxSkeleton class="onyx-accordion-item-skeleton__main" />
+    <OnyxSkeleton class="onyx-accordion-item-skeleton__decoration" />
+  </div>
+
+  <details
+    v-else-if="accordionContext"
+    class="onyx-component onyx-accordion-item"
+    :class="[densityClass, props.disabled || accordionContext.disabled.value ? 'disabled' : '']"
+    :open="isOpen"
+    @toggle="(e) => (isOpen = (e.target as HTMLDetailsElement).open)"
+  >
+    <summary
+      class="onyx-accordion-item__header"
+      :tabindex="props.disabled || accordionContext.disabled.value ? -1 : 0"
+    >
+      <slot name="header"></slot>
+      <OnyxIcon :icon="chevronLeftSmall" class="onyx-accordion-item__header__icon" />
+    </summary>
+    <div class="onyx-accordion-item__panel">
+      <slot name="panel"></slot>
+    </div>
+  </details>
+  <!-- Fallback if AccordionItem is used without Accordion -->
+  <details
+    v-else
+    class="onyx-component onyx-accordion-item"
+    :class="[densityClass, props.disabled ? 'disabled' : '']"
+  >
+    <summary class="onyx-accordion-item__header" :tabindex="props.disabled ? -1 : 0">
+      <slot name="header"></slot>
+      <OnyxIcon :icon="chevronLeftSmall" class="onyx-accordion-item__header__icon" />
+    </summary>
+    <div class="onyx-accordion-item__panel">
+      <slot name="panel"></slot>
+    </div>
+  </details>
+</template>
+
+<style lang="scss">
```

Comment:
Could you please check the styles with Figma? There are currently a few differences like text color, font weight etc.

Also, I'd prefer to use a flex layout instead of positioning the chevron icon absolute